packages/xstate-solid/src/useActor.ts

```diff
@@ -40,25 +38,25 @@ export function useActor(
   });
 
   // Track if a new actor is passed in, only run once per actor
-  createEffect(
+  createRenderEffect(
```

Comment:
I wonder how hard it would be to add a test for the `createEffect` -> `createRenderEffect`? Would you be able to craft one?